impress-org/givewp
# Feature: Donation table Live/Test switch

## Description

This PR adds a toggle component to the donations table so that users can query between Live Donations and Test donations. It is initialized by the users paymentgateway mode and updates when the setting is changed. The setting preference is passed down to the table via window.GiveDonations

## Affects

DonationListTable

## Visuals
<img width="1723" alt="Screen Shot 2022-11-21 at 8 07 24 AM" src="https://user-images.githubusercontent.com/75056371/203178649-016a57cc-d7b1-4dfb-be5e-e2776658f9c0.png">

## Testing Instructions

- Visit the Donations table.
- Toggle the settings and verify the correct data is loaded.

## Pre-review Checklist
